docs/matching-content-to-subscriber-lists.md

```diff
@@ -27,6 +24,11 @@ similar to the links hash on a content item. For example:
 }
 ```
 
+Only one of these should be used on a given subscriber list; `tags` is
+deprecated. They can also both be empty `{}` if they aren't needed.
```

Comment:
We should probably drop the terminology about tags being deprecated. They don't seem to be going anywhere and they're the only thing we've got to inform about emails that is not related to a content item (which seems to be an ongoing need).

If it's not opening a fresh can of worms would be able to quietly remove the deprecation reference?

Reply:
Sure, that makes sense - it's used by the Brexit Checker. I've added another commit.

Reply:
Thanks 👍